package.json

```diff
@@ -8,14 +8,14 @@
   "homepage": "https://github.com/webpack/schema-utils",
   "bugs": "https://github.com/webpack/schema-utils/issues",
   "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "types": "declarations/index.d.ts",
   "engines": {
     "node": ">= 8.9.0"
   },
   "scripts": {
     "start": "npm run build -- -w",
     "prebuild": "npm-run-all --serial clean build:types",
-    "build:types": "tsc --declaration --emitDeclarationOnly --outDir dist",
+    "build:types": "rimraf declarations && tsc --declaration --emitDeclarationOnly --outDir declarations && prettier \"declarations/**/*.ts\" --write",
```

Comment:
You probably want to check as part of the CI process if the declarations are up-to-date. So contributors do not forget to build the declarations

Reply:
You are correct, but I can not figure it out how to do it better, looks like for webpack core repo it also can be a problem (if declarations and JSON schemas will be created from code)

Maybe add some *special* step in CI where we bash script aka:
```
npm run build
if [ -n "$(git status --porcelain)" ]; then
  echo "Missing declarations in git";
  exit 1;
else
  echo "All declarations are valid";
fi
```

So if this step fails contributor can get some info about it from `webpack-bot` +
maybe someone wants to implement autocommit for `webpack bot` =)

Reply:
/cc @evilebottnawi what do you think about it?

Reply:
@vankop let's solve this in other PR, can you open issue?